Test/Utils/ContrailAPI/ConfigureDNS.Tests.ps1

```diff
@@ -0,0 +1,132 @@
+Param (
+    [Parameter(Mandatory=$false)] [string] $TestenvConfFile = "C:\scripts\configurations\test_configuration.yaml",
+    [Parameter(ValueFromRemainingArguments=$true)] $UnusedParams
+)
+
+. $PSScriptRoot\ConfigureDNS.ps1
+. $PSScriptRoot\..\ContrailUtils.ps1
+. $PSScriptRoot\..\..\..\CIScripts\Testenv\Testenv.ps1
+
+
+# TODO: Those tests run on working Controller.
+#       Most probably they need to be rewrote
+#       to use some fake.
+# NOTE: Because of the above they should not be run automatically
+Describe 'Configure DNS API' {
+    BeforeAll {
+        $OpenStackConfig = Read-OpenStackConfig -Path $TestenvConfFile
+        $ControllerConfig = Read-ControllerConfig -Path $TestenvConfFile
+
+        [Diagnostics.CodeAnalysis.SuppressMessageAttribute(
+            "PSUseDeclaredVarsMoreThanAssignments",
+            "ContrailUrl",
+            Justification="It's actually used."
+        )]
+        $ContrailUrl = $ControllerConfig.RestApiUrl()
+
+        [Diagnostics.CodeAnalysis.SuppressMessageAttribute(
+            "PSUseDeclaredVarsMoreThanAssignments",
+            "AuthToken",
+            Justification="It's actually used."
+        )]
+        $AuthToken = Get-AccessTokenFromKeystone `
+            -AuthUrl $OpenStackConfig.AuthUrl() `
+            -Username $OpenStackConfig.Username `
+            -Password $OpenStackConfig.Password `
+            -Tenant $OpenStackConfig.Project
+    }
+
+    It 'adds and removes "empty" contrail DNS server' {
+        $ServerUUID = Add-ContrailDNSServer -ContrailUrl $ContrailUrl -AuthToken $AuthToken `
+            -DNSServerName "CreatedByPS1Script"
+        Remove-ContrailDNSServer -ContrailUrl $ContrailUrl -AuthToken $AuthToken -DNSServerUuid $ServerUUID
+    }
```

Comment:
This test (and some others) don't assert anything. If we expect that something should not throw, then we should probably use `Should -Not -Throw` assert.